### DISCUSSION TITLE
Step 1 Completed - Continuous Integration (CI)

### DISCUSSION THREAD

**Discussion by mlaustin11** [skills/.github#780] [Test with Actions]
Hello, 
I went through and completed step 1 but I'm stuck on Step 2.  I didn't have any actions to Fix, and didn't receive red X.  It completed Successfully in Step 1 with no issues.  Have I done something wrong?  I'm new to this.  Any assistance would be great!  Thank You.

**Comment by d121122gma** [comment]
> Hello, 
> 
> I went through and completed step 1 but I'm stuck on Step 2.  I didn't have any actions to Fix, and didn't receive red X.  It completed Successfully in Step 1 with no issues.  Have I done something wrong?  I'm new to this.  Any assistance would be great!  Thank You.

**Comment by d121122gma** [comment]
> Hello, 
> 
> I went through and completed step 1 but I'm stuck on Step 2.  I didn't have any actions to Fix, and didn't receive red X.  It completed Successfully in Step 1 with no issues.  Have I done something wrong?  I'm new to this.  Any assistance would be great!  Thank You.
